app/plot_app/configured_plots.py

```diff
@@ -186,19 +193,97 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data, link_to_3d_page,
 
     if data_plot.finalize() is not None: plots.append(data_plot)
 
-
+    # VTOL tailistter orientation conversion, if relevant
+    if is_vtol_tailsitter:
+        print(vtol_states)
```

Comment:
Can you remove the print and move this into a separate method in a new file `vtol_tailsitter.py`?